requirements_maint.txt

```diff
@@ -1,3 +1,3 @@
-pygithub~=2.4
+pygithub~=2.5
```

Comment:
I do not think it is needed to force a newer version. We already allow everything `2.4 <= x < 3` ...